functions/__tests__/functions.test.ts

```diff
@@ -42,6 +42,7 @@ describe('extension', () => {
   beforeEach(() => {
     restoreEnv = mockedEnv(defaultEnvironment)
     config = require('../src/config').config
+    config.collectionPath = 'collection'
```

Comment:
Is this new? What is it for?

Reply:
Not sure why it has been added. @oplik0 could maybe tell us more.

Reply:
This is a result of a change in firebase functions event triggers (line 37) - `handler` was removed and instead one needs to use `document(path).onWrite`.

With the regex for the collection path (I think it only allowed odd number of path parts - so only paths to a collection and not a specific document) this should always result in a wildcard path for all documents in selected collection.

Tge tests here didn't have a collection path despite it technically being required by the extension, and while it worked before now the path is used in the listener, so it had to be set in the config for these tests.